### PR TITLE
Procedures, Tabs, Tables

### PR DESCRIPTION
**Pull request by nathan-contino-mongo**
[Procedures](https://docs-mongodbcom-staging.corp.mongodb.com/meta/docsworker-xlarge/feature-skunkworks/reference/procedures/)
[Tabs](https://docs-mongodbcom-staging.corp.mongodb.com/meta/docsworker-xlarge/feature-skunkworks/reference/tabs/)
[Tables](https://docs-mongodbcom-staging.corp.mongodb.com/meta/docsworker-xlarge/feature-skunkworks/reference/tables/)

### CODE REVIEW

File: source/reference/tabs.txt [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+~~~~~~~~~~~
+
+You can add the ``:hidden:`` option to any pre-configured or custom tabs
+directive to hide the tab titles and tab selector GUI on that tabset.
+This can be useful when you want to offer a tab option at the top of a
+page, but that option isn't useful further down the page. For example,
+a tutorial with multiple sequential steps (some common between
+environments, some tabbed) could benefit from hiding tabs after the first
+step, since users aren't likely to switch between tabs in the middle of
+the tutorial.
```
Comment (kyuan-mongodb) [inline source/reference/tabs.txt:113]: Suggestion: Similar to the comment in the Procedures Page -- can an example from a docs repo be linked here?

File: source/reference/procedures.txt [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+          :copyable: false
+     - .. include:: /includes/procedure-example-connected.rst
+
+   * - .. literalinclude:: /includes/procedure-example-normal.rst
+          :copyable: false
+     - .. include:: /includes/procedure-example-normal.rst
+
+   * - .. literalinclude:: /includes/procedure-example-compact.rst
+          :copyable: false
+     - **compact style does not render on non-landing pages**
```
Comment (kyuan-mongodb) [inline source/reference/procedures.txt:55]: Suggestion: Can an example from a docs repo be linked here? This left me wondering what it actually looks like when on a landing page.

File: source/reference/tables.txt [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. include:: /includes/styles/corrections.rst
+
+Tables compare items across multiple domains.
```
Comment (kyuan-mongodb) [inline source/reference/tables.txt:17]:
```suggestion
Tables help compare items across multiple domains or present information in a ___ manor.
```
Suggestion: I think tables are also used a lot when describing parameters for an API or function -- so I think its worth incorporating that here.

File: source/reference/tables.txt [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+Table Types
+-----------
+
+rST offers multiple kinds of table, but ``list-table`` is the only
+variant we use at MongoDB.
+
+Introduce Tables
+----------------
+
+Tables represent lots of information simultaneously. To help your readers
```
Comment (kyuan-mongodb) [inline source/reference/tables.txt:29]: Suggestion: Somewhere on this page (possibly here), I think you should explicitly callout that you can have any number of rows and columns and how each group of bullets after need to match that (even if some are blank).

**Review by kyuan-mongodb — CHANGES_REQUESTED** [review]
This is amazing! Left some suggestions and questions I'd like to be considered or addressed before merging.

File: source/reference/procedures.txt [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
```
[the remark this answers is not in the document]
Comment (nathan-contino-mongo) [reply source/reference/procedures.txt:55]: Unfortunately I can't find any use of it -- even the procedures on the Cloud landing page appear to be `connected` style 😢

File: source/reference/procedures.txt (hunk 2 above)
Comment (nathan-contino-mongo) [reply source/reference/procedures.txt:55]: Looks like this isn't in use anywhere yet -- but I added a link to a Realm SDK docs example procedure use in practice.

File: source/reference/tables.txt (hunk 3 above)
Comment (nathan-contino-mongo) [reply source/reference/tables.txt:17]: Added in my own commit!

File: source/reference/tables.txt (hunk 4 above)
Comment (nathan-contino-mongo) [reply source/reference/tables.txt:29]: Added during the discussion of row formatting.

**Pull request merged**